command/connect/envoy/envoy.go

```diff
@@ -165,6 +167,19 @@ func (c *cmd) init() {
 			"consul.destination.[service|dc|...]. The old tags were preserved for backward compatibility,"+
 			"but can be disabled with this flag.")
 
+	c.flags.StringVar(&c.prometheusBackendPort, "prometheus-backend-port", "",
```

Comment:
I believe there should also be `website` updates to the [CLI section](https://www.consul.io/commands/connect/envoy) and possibly something else (?).

Reply:
I added it to that CLI section and looked around for any other areas these flags are mentioned and didn't see any. There will be a future docs PR for the overall metrics feature in Consul-k8s that should hopefully cover anything else.